index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <title>Fake News Detection</title>
+</head>
+
+<body>
+    <div class="container mt-5">
+        <h1>Fake News Detector</h1>
+        <form id="newsForm">
+            <div class="form-group">
+                <label for="title">Article Title:</label>
+                <input type="text" class="form-control" id="title" required>
+            </div>
+            <div class="form-group">
+                <label for="text">Article Text:</label>
+                <textarea class="form-control" id="text" rows="5" required></textarea>
+            </div>
+            <button type="submit" class="btn btn-primary">Detect</button>
+        </form>
+        <div id="result" class="mt-4"></div>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script>
+        $("#newsForm").submit(function (e) {
+            e.preventDefault();
+            const title = $("#title").val().trim();
+            const text = $("#text").val().trim();
+            if (title === "" || text === "") {
+                alert("Please enter both the title and text of the article.");
+                return;
+            }
+
+            $.ajax({
+                url: "http://localhost:8000/detect",
+                type: "POST",
+                data: JSON.stringify({ title: title, text: text }),
+                contentType: "application/json",
+                success: function(response) {
+                    const result = response.detection;
+                    const verdict = result > 0.5 ? "Fake" : "False"
+                    $("#result").html(`<h4>Verdict: <span class="badge badge-primary">${verdict}</span></h4>`);
+                },
+            });
+
+            $("#result").html(`<h4>Verdict: <span class="badge badge-primary">Amidst the digital storm, the detector contemplates its decision with epic solemnity...</span></h4>`);
```

Comment:
I think `verdict` isn't an appropriate word, change to 'result' maybe or smth like that, or just omit this and have only this `${verdict}` value

Reply:
i wanted it to sound epic and cool😣

Reply:
I'm sorry, then you can leave it as it is. I'm very sorry I've made you feel bad(

Reply:
i fixed it already so don't worry ƪ(˘⌣˘)ʃ

Reply:
Oh nooooooooo